src/core.ts

```diff
@@ -64,8 +64,9 @@ export class TemplateResult {
       // We're in a text position if the previous string closed its tags.
       // If it doesn't have any tags, then we use the previous text position
       // state.
-      const closing = findTagClose(s);
-      isTextBinding = closing > -1 ? closing < s.length : isTextBinding;
+      const close = s.lastIndexOf('>');
+      const open = s.indexOf('<', close + 1);
+      isTextBinding = open === -1 && (close > -1 || isTextBinding);
```

Comment:
Let's add a little comment visualization if possible. Not sure this helps but something like:
```
// when string ends with:
// text: ... > ... 
// not text: ... < ...
// take last value: ... or ... < ...
```

Reply:
Does flipping it around work and make it smaller?
`isAttributeBinding = open > -1 || isAttributeBinding;`

Reply:
I made a truth table and did the logical reduction, so I think this is minimal.

Reply:
Updated the comment